drivers/led/Kconfig.max7219

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2022 Jimmy Ou <yanagiis@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig MAX7219
+	bool "MAX7219 LED drivers"
+	depends on SPI
```

Comment:
Add a `default $(dt_compat_enabled,$(DT_COMPAT_MAXIM_MAX7219))`

(see `drivers/sensor/lm75/Kconfig` for an example)

Reply:
OK. Thanks. I will add that.